tutor/src/screens/assignment-edit/ux.js

```diff
@@ -65,7 +65,6 @@ export default class AssignmentUX {
       }
     }
     this.course = course;
-    await this.course.referenceBook.ensureLoaded();
     this.dueAt = due_at;
```

Comment:
it doesn't hurt to call `ensureLoaded`, it shouldn't do anything if the model was already fetched.

Reply:
👍